### ISSUE TITLE
apr_epic story 1: qual_workflow_maps

### ISSUE THREAD

**Issue by jessfroe** [lzim/teampsd#1394]
Creation of flow maps to document and disseminate knowledge of qual_workgroup processes. LucidCharts will be used to create maps.

**Comment by lzim** [comment]
Thanks @jessfroe for starting this card!
🤗
I’ve removed the rest of **qual_workgroup** assignees, because I believe you’re the one working in this, right? We only want one responsible “assignee” and the label denote the workgroup “owner” making sure this gets done and isn’t blocking or blocked by anything else. 

If this is the master **document_card** to track the requirements for the document, then there would be another **issue_task_card** to track the estimated effort to complete this. 

Also, I see the 6/3 due date, is there any way we could work to get these done in the **may_epic?** 
We are trying to **finish #1192** & **close all open document cards** in May. 🎢

@lijenn Can you keep a close eye on observing  Jessilyn’s user experience, as she works on this to see what she does/does not access, or does/does not understand, from our training resources?
Jessilyn, in May we’re trying to get to a place where our resources are so self-explanatory, that any new Team PSD members would be able to do this 100% on their own using the Team PSD repo.

**Comment by lijenn** [comment]
@jessfroe I apologize for the late response! You had some great questions in the email that I'd love to answer here so it's easier to keep track of :)!

1. Should a general issue card be triaged in the document_tracker?
Let's define what you meant by general issue card.
- Yes - If you meant that it is a card that will contain and track **all the tasks** needed to be completed (let's call this the **Master Document card**).
- No - if it is just the card containing **the single work/task itself** (let's call these **issue task card(s)**).

Based off of the description in your top post (I've also removed the prior body template as it is not needed!) it sounds like this is a Master Document Card. If so, you are correct that it would go in the document_tracker. Let's triage it to **document_teams** as well since it would apply to the Qual workgroup's workflow and team members' (old or new) needs. 

2. Should each map have it's own separate issue card?
Yes! These would considered issue task cards. You can use this Master Document card to keep track of each map's issue card. 

3. # of maps needed
For this, I think it makes sense to see what is already available that may reduce your and/or Qual's workload in creating maps. I think it would also help to know what you guys are aware of or what in general seems confusing. 

Here is the ZenHub flow map too, I forgot to include this in the email!
https://github.com/lzim/teampsd/blob/master/resources/maps/zenhub_flow.pdf

**Comment by lzim** [comment]
- Define our repository with code while people learn how to build out a repo.
- We'll build tests for our standards, then throw new things against the tests.
- Remove reliance on team huddles.
- Define a "CSS" like style on every file on the repo conforms to.
- Define the data dictionary for the flow of data from CDW through to the model or to R.
- Automated documentation of changes.
- Triage and disposition changes against standards and have them carry (refactor) through all instances.
- Knew how to test my own file against the standard a) style and b) configuration.
- Alerted when code changes that I should know about in my own workflow.

**Comment by lzim** [comment]
@staceypark 
@jessfroe 
@teampsdkathryn 
@dlounsbu 
@swapmush 

After we get the manuscript out.
Let's revisit making sure our workflow is clear in relation to **Story 1** Standard Research!

**Comment by lzim** [comment]
Stale.
